CMSgov/design-system
# [WNMGDS 2915] Remove doc site email icon

## Summary

- [Ticket](https://jira.cms.gov/browse/WNMGDS-2915)
- Deleted doc site specific EmailIcon component
- Removed Email section from Contact Page on Doc site since it appears to be commented out and we don't have an email address in the section.

## How to test

1. Run browser tests locally, they should all pass
2. Confirm the Utility Actions page (under /patterns) looks like the attached screenshot. The Email Icon should be filled and spaced correctly.
3. Confirm the Contact page looks like the attached screenshot. The Email section should be deleted.

## Checklist


### Updated Utility Actions page
<img width="686" alt="Screenshot 2024-09-20 at 3 14 03 PM" src="https://github.com/user-attachments/assets/153de290-cc6c-4118-b2f2-40b8ba3df4b4">

### Updated Contact page
<img width="717" alt="Screenshot 2024-09-20 at 3 14 19 PM" src="https://github.com/user-attachments/assets/40c2186e-f369-4fdd-8561-917a8965578a">

### Previous Contact page
<img width="727" alt="Screenshot 2024-09-20 at 3 15 19 PM" src="https://github.com/user-attachments/assets/a8c688f6-f64a-4e31-b9bd-ace1376066de">
